src/OmniSharp.Host/WorkspaceInitializer.cs

```diff
@@ -24,9 +27,11 @@ public class WorkspaceInitializer
 
             var projectEventForwarder = compositionHost.GetExport<ProjectEventForwarder>();
             projectEventForwarder.Initialize();
+            var projectSystems = compositionHost.GetExports<Lazy<IProjectSystem, ProjectSystemMetadata>>();
```

Comment:
maybe we should hide that logic in some generic extension method? this way it could be reused in the future against other potentially ordered look ups like i.e. https://github.com/OmniSharp/omnisharp-roslyn/blob/5129ba5a7b5b401935cf234b32a45611bb1e3b13/src/OmniSharp.Host/WorkspaceInitializer.cs#L74